exampleSpecs/scatter.json

```diff
@@ -1,9 +1,9 @@
 {
-  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
```

Comment:
Why remove the schema?

Reply:
I just blindly copy and paste from issue. Will fix it.